### DISCUSSION TITLE
How to use global model?

### DISCUSSION THREAD

**Discussion by wanderine** [NVIDIA/NVFlare#1920] [Q&A] [answered]
In the examples I cannot see how to use the resulting global model. Where is it stored? Is it under /tmp/nvflare/jobs-storage/.../data ? It seems to be a single file. Does it contain the model for all rounds? How do I load the model from each round?

**Comment by chesterxgchen** [comment, marked as answer]
the global model is broadcast to each client during each round for Scatter and Gather (SAG) workflow. The resulting model is stored at the server workspace (with model persistor) at the server location (if you using SAG), client can use download model command to get the model from server after the training

**Reply by chatzikon** [reply]
How can I use the download model command in clients? I couldn't find something at the nvflare documentation.
